source/_dashboards/humidifier.markdown

```diff
@@ -32,6 +32,10 @@ theme:
   required: false
   description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
   type: string
+current_humidity_sensor:
+  required: false
+  description: Sensor for the current humidity. Overrides current_humidity attribute.
+  type: string
```

Comment:
I think this should be removed completely, as it is out of "wack" with how all our other cards work.

Now the `current_humidity` is part of the entity component, we don't need this anymore.

../Frenck

Reply:
Because this is the only change, I will just close this PR.